### DISCUSSION TITLE
Auto Refresh

### DISCUSSION THREAD

**Discussion by mrtaufiq00** [dbeaver/dbeaver#22519] [Q&A]
does someone help me?
How to remove warn "Confirm a possible data modification" on dbeaver when i start auto refresh 
![WhatsApp Image 2024-01-13 at 4 54 55 PM](https://github.com/dbeaver/dbeaver/assets/157184412/107eab4d-df99-4412-925f-f13f14966965)

**Comment by E1izabeth** [comment]
Currently there is no way to avoid this confirmation.

**Reply by mrtaufiq00** [reply]
thankyou @E1izabeth
